data/json/mutations/mutations.json

```
@@ -2907,7 +2907,7 @@
     "prereqs": [ "ROOTS2" ],
     "prereqs2": [ "LEAVES" ],
     "threshreq": [ "THRESH_PLANT" ],
-    "cancels": [ "HERBIVORE", "CARNIVORE", "GRAZER" ],
+    "cancels": [ "CARNIVORE" ],
```

```suggestion
    "cancels": [ "GRAZER" ],
```
Geting nutritions from rotting meat or vegetable makes sense, however I'm not sure it makes sense to be both a grazer and a saprophage since this assumes you're a post thresh plant mutant.

---

done :)